### ISSUE TITLE
Svg file Support

### ISSUE THREAD

**Issue by lowbouncerate** [ribtoks/xpiks#802]
SVG files are not supported. 
Look like it is not reading at all

**Comment by ribtoks** [comment]
Yes, SVG files are not supported. Which microstock requires to send SVG?
